src/content/getting-started/custom-taxonomies.mdx

```diff
@@ -18,6 +18,9 @@ Custom Post Type and enabling GraphQL support.
 ```php
 add_action('init', function() {
   register_taxonomy( 'doc_tag', 'docs', [
+    'labels'  => [
+	'menu_name' => 'Document Tags'
```

Comment:
I'm fine with this for the example, but we should also wrap in translation function as well: `__( 'Document Tags', 'your-textdomain' ),`

Reply:
@jasonbahl good point but like in the other PR, my point was by default WordPress will consider the cpt as page and taxonomy as tags. I've added i18n stuffs

Reply:
Thanks!